tests/Application/Actions/Donations/UpdateHandlesLockExceptionTest.php

```diff
@@ -9,6 +9,7 @@
 use Doctrine\ORM\EntityManagerInterface;
 use GuzzleHttp\Psr7\ServerRequest;
 use MatchBot\Application\Actions\Donations\Update;
+use MatchBot\Client\Stripe;
 use MatchBot\Domain\Campaign;
 use MatchBot\Domain\Charity;
 use MatchBot\Domain\Donation;
```

Comment:
this test gets quite a bit simpler as a nice side effect of the work to support load testing.